stubs/click-shell/click_shell/core.pyi

```diff
@@ -0,0 +1,34 @@
+from logging import Logger
+from typing import Any, Callable
```

Comment:
[precommit.ci](https://results.pre-commit.ci/run/github/31696383/1755262846.2AYN4TsST9Ws7NULx8rKkA) already reported this but we use `Callable` from `collections.abc`
```suggestion
from collections.abc import Callable
from typing import Any,

```